app/src/main/java/com/liveramp/ats/sampleapp/MainActivity.java

```diff
@@ -62,14 +55,15 @@ protected void onCreate(Bundle savedInstanceState) {
     // Example on how to init LiveRamp
     private void initializeLiveRampATS(){
 
+
         if (notCheckingForCCPA || supportOtherGeos) {
-            LRAtsManager.INSTANCE.hasConsentForNoLegislation();
+            LRAtsManager.INSTANCE.setHasConsentForNoLegislation(true);
         }
 
-        // Notice: This will be removed in a subsequent SDK release
-        Configuration fallbackConfig = generateFallBackConfigurationForFileName("fallback_configuration.json");
-
-        LRAtsConfiguration config = new LRAtsConfiguration(appID, fallbackConfig);
+        // For the 2nd param (boolean), set "true" to enable test mode
+        // In test mode - the SDK will "simulate" envelopes.s
+        // Note: No network calls will actually be made!
+        LRAtsConfiguration config = new LRAtsConfiguration(appID, true);
```

Comment:
Note for the public: setting this to `true` will cause the SDK not perform any network requests. Remember to set this to `false` before going to production!